### ISSUE TITLE
Add ARIA label to highlight template markup

### ISSUE THREAD

**Issue by nevstokes** [gohugoio/hugo#7526]
On the back of #7194 and to further improve accessibility in Hugo, it should be possible to add an `aria-label` attribute to code blocks generated through the `{{ highlight }}` shortcode or through the use of Markdown code fences.

For example `aria-label="Code sample"`
